source/patterns/@aws-solutions-constructs/aws-lambda-kendra/README.md

```diff
@@ -0,0 +1,133 @@
+| deployVpc? | `boolean` | Whether to create a new VPC based on `vpcProps` into which to deploy this pattern. Setting this to true will deploy the minimal, most private VPC to run the pattern:<br>\- One isolated subnet in each Availability Zone used by the CDK program<br>\- `enableDnsHostnames` and `enableDnsSupport` will both be set to true<br>If this property is `true` then `existingVpc` cannot be specified. Defaults to `false`. |
+
+## Pattern Properties
+
+| **Name** | **Type** | **Description** |
+| --- | --- | --- |
+| lambdaFunction | [`lambda.Function`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.Function.html) | Returns an instance of `lambda.Function` managed by the construct |
+| kendraIndex | [`kendra.cfnIndex`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_kendra.CfnIndex.html) | Returns an instance of `kendra.cfnIndex` managed by the construct |
+| kendraDataSources | [`interface DataSourceProperties {
+[]`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_kendra.CfnDataSource.html) | A list of data sources created for this construct/index, each in an object that includes the role for that data source. |
+| lambdaRole | [`iam.Role`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_iam.Role.html) | The role assumed by the Lambda function |
+| vpc? | [`ec2.IVpc`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.IVpc.html) | Returns an interface on the VPC used by the pattern (if any). This may be a VPC created by the pattern or the VPC supplied to the pattern constructor. |
+
+interface DataSourceProperties {
+  role?: [`iam.Role`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_iam.Role.html),
+  source: | [`CfnDataSource`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_kendra.CfnDataSource.html)
+}
+## Lambda Function
+
+This pattern requires a lambda function that can access a Kendra index.
+
+## Default settings
+
+Out of the box implementation of the Construct without any overrides will set the following defaults:
+
+### AWS Lambda Function
+
+- Configure limited privilege access IAM role for Lambda function
+- Enable reusing connections with Keep-Alive for Node.js Lambda function
+- Enable X-Ray Tracing
+- Set Environment Variables
+    - (default) KENDRA_INDEX_ID
+    - AWS_NODEJS_CONNECTION_REUSE_ENABLED
+
+### Amazon Kendra Index
+
+- Creates Amazon Kendra endpoint in VPC if appropriate
+- Defaults to DEVELOPER_EDITION
+
+**Amazon Kendra DataSources**
+
+- Sets up correct IAM roles to access data for:
+    - S3 data sources
+    - Which others should we support in MLP? https://docs.aws.amazon.com/kendra/latest/dg/iam-roles.html
+- Adds each data source to Kendra index
+
+## Architecture
+
+![Architecture Diagram](/Applications/Joplin.app/Contents/Resources/app.asar/architecture.png)
```

Comment:
I don't see an actual architecture.png commited in this PR, and not sure about the URL that is specified.